### PR TITLE
OpcodeDispatcher: Optimize phminposuw

### PR DESCRIPTION
**Pull request by Sonicadvance1**
I would now consider the XMM version of this to be optimal.

Thanks to @rygorous for giving the idea for how to optimize this!

### CODE REVIEW

File: FEXCore/include/FEXCore/IR/IR.h [hunk 1]
```diff
@@ -513,6 +513,12 @@ enum class SyscallFlags : uint8_t {
 
 FEX_DEF_NUM_OPS(SyscallFlags)
 
+enum NamedVectorConstant : uint8_t {
```
Comment (lioncash) [inline FEXCore/include/FEXCore/IR/IR.h:516]: Perhaps add a comment that this is linked to the array in CPUBackend.cpp as well and what it's for

File: FEXCore/Source/Interface/IR/IRDumper.cpp [hunk 2]
```diff
@@ -183,6 +183,15 @@ static void PrintArg(fextl::stringstream *out, [[maybe_unused]] IRListView const
   }
 }
 
+static void PrintArg(fextl::stringstream *out, [[maybe_unused]] IRListView const* IR, FEXCore::IR::NamedVectorConstant Arg) {
+  switch (Arg) {
+    case FEXCore::IR::NamedVectorConstant::NAMED_VECTOR_INCREMENTAL_U16_INDEX: *out << "u16_incremental_index"; break;
```
Comment (lioncash) [inline FEXCore/Source/Interface/IR/IRDumper.cpp:188]: Considering how long these identifiers are, maybe line break the body onto a new line

File: FEXCore/Source/Interface/IR/IR.json [hunk 3]
```diff
@@ -1028,6 +1029,11 @@
         "NumElements": "RegisterSize / ElementSize"
       },
 
+      "FPR = LoadNamedVectorConstant u8:#RegisterSize, NamedVectorConstant:$Constant": {
+        "Desc": ["Load a named vector constant"
```
Comment (lioncash) [inline FEXCore/Source/Interface/IR/IR.json:1033]: Maybe indicate where to find the available list of constants here as well

**Review by lioncash — COMMENTED** [review]
Looks good. Just a few nits

File: FEXCore/include/FEXCore/IR/IR.h (hunk 1 above)
Comment (Sonicadvance1) [reply FEXCore/include/FEXCore/IR/IR.h:519]: Added a comment.

File: FEXCore/Source/Interface/IR/IRDumper.cpp (hunk 2 above)
Comment (Sonicadvance1) [reply FEXCore/Source/Interface/IR/IRDumper.cpp]: Split between lines.

File: FEXCore/Source/Interface/IR/IR.json (hunk 3 above)
Comment (Sonicadvance1) [reply FEXCore/Source/Interface/IR/IR.json]: Added a comment where to find them.

**Review by lioncash — APPROVED** [review]

**Pull request merged**